ravenpy/models/emulators/canadianshield.py

```diff
@@ -0,0 +1,616 @@
+        #        :Units,          frac,          frac,
+        #   FOREST,                0.0,       0.02345 ## forest_frac,
+        # :EndLandUseClasses
+        {land_use_classes}
+
+        #-----------------------------------------------------------------
+        # Vegetation Classes
+        #-----------------------------------------------------------------
+        :VegetationClasses,
+                :Attributes,            MAX_HT, MAX_LAI, MAX_LEAF_COND,
+                :Units,                      m,    none,      mm_per_s,
+                FOREST,                    5.0,     5.0,           5.0,
+        :EndVegetationClasses
+
+        #-----------------------------------------------------------------
+        # Global Parameters
+        #-----------------------------------------------------------------
+        :GlobalParameter          SNOW_SWI par_x15            # para_x15
+        :GlobalParameter      SNOW_SWI_MIN par_x16            # para_x16
+        :GlobalParameter      SNOW_SWI_MAX par_sum_x16_x17    # para_x16+para_x17
+        :GlobalParameter  SWI_REDUCT_COEFF par_x18            # para_x18
+        :GlobalParameter     RAINSNOW_TEMP par_x19            # para_x19
+        :GlobalParameter    RAINSNOW_DELTA par_x20            # para_x20
+        :GlobalParameter   MAX_SWE_SURFACE par_x21            # para_x21
+        :GlobalParameter    TOC_MULTIPLIER par_x22            # para_x22
+
+        #-----------------------------------------------------------------
+        # Soil Parameters
+        #-----------------------------------------------------------------
+        :SoilParameterList
+          :Parameters,        POROSITY,  HBV_BETA,  BASEFLOW_COEFF,    BASEFLOW_N,  MAX_INTERFLOW_RATE,   FIELD_CAPACITY,   SAT_WILT,   MAX_PERC_RATE,  PET_CORRECTION,
+               :Units,               -,         -,               -,             -,                   -,                -,          -,               -,               -,
+              TOPSOIL,             1.0,   par_x07,             0.0,           0.0,             par_x12,  par_sum_x05_x06,    par_x05,         par_x13,         par_x04,
+               VADOSE,             1.0,       0.0,     par_pow_x08,       par_x10,                 0.0,              0.0,        0.0,         par_x14,             0.0,
+          FRACBEDROCK,             1.0,       0.0,     par_pow_x09,       par_x11,                 0.0,              0.0,        0.0,             0.0,             0.0,
+        #     TOPSOIL,             1.0,  para_x07,             0.0,           0.0,            para_x12, para_sum_x05_x06,   para_x05,        para_x13,        para_x04,
+        #      VADOSE,             1.0,       0.0,    para_pow_x08,      para_x10,                 0.0,              0.0,        0.0,        para_x14,             0.0,
+        # FRACBEDROCK,             1.0,       0.0,    para_pow_x09,      para_x11,                 0.0,              0.0,        0.0,             0.0,             0.0,
```

Comment:
Do we need to keep those commented lines ?

Reply:
I would leave them in. Otherwise, it is really hard for the user to identify which parameter is which once the values get filled in. The commented lines won't get filled in because they contain, for example, `para_x07` and not `par_x07`.

Reply:
You're referring to the "filling-in" by Ostrich ?

Reply:
No, also in general. 

If the user downloads the RV* files with the parameters set he/she might have chosen. There would be no way for the user to know where parameter x7 is located in those files when you remove those comment lines.

Reply:
I see your point. Makes sense.